components/widgets/climate/carbon-flux/selectors.js

```diff
@@ -0,0 +1,273 @@
+                return (
+                  <g transform={`translate(0 ${height / 2 + offset})`}>
+                    <text
+                      x={
+                        emissionsData > 0
+                          ? x + width + offset
+                          : x + width - offset
+                      }
+                      y={y}
+                      textAnchor={emissionsData > 0 ? 'start' : 'end'}
+                      fill="#000"
+                      fontSize={14}
+                    >
+                      EMISSIONS
+                    </text>
+                  </g>
+                );
+              },
+            },
+          },
+          removals: {
+            fill: removals,
+            background: false,
+            stackId: 2,
+            labelList: {
+              content: (props) => {
+                // eslint-disable-next-line react/prop-types
+                const { x, y, offset, width, height } = props;
+                return (
+                  <g transform={`translate(0 ${height / 2 + offset})`}>
+                    <text
+                      x={
+                        removalsData > 0
+                          ? x + width + offset
+                          : x + width - offset
+                      }
+                      y={y}
+                      textAnchor={removalsData > 0 ? 'start' : 'end'}
+                      fill="#000"
+                      fontSize={14}
+                    >
+                      REMOVALS
+                    </text>
+                  </g>
+                );
+              },
+            },
+          },
+          flux: {
+            fill: netFluxData > 0 ? netEmissions : netRemovals,
+            background: false,
+            labelList: {
+              content: (props) => {
+                // eslint-disable-next-line react/prop-types
+                const { x, y, width, height, offset } = props;
+                return (
+                  <g transform={`translate(0 ${height / 2 + offset})`}>
+                    <text
+                      x={
+                        netFluxData > 0
+                          ? x + width + offset
+                          : x + width - offset
+                      }
+                      y={y}
+                      textAnchor={netFluxData > 0 ? 'start' : 'end'}
+                      fill="#000"
+                      fontSize={14}
+                    >
+                      {netFluxData > 0 ? 'NET EMISSIONS' : 'NET REMOVALS'}
+                    </text>
+                  </g>
+                );
+              },
+            },
+          },
+        },
+      },
+      referenceLine: { x: 0, stroke: '#606060', strokeWidth: 2 },
+      tooltip: [
+        {
+          label: 'Carbon flux (per year)',
+        },
+        {
+          key: 'removals',
+          label: 'Removals',
+          unitFormat: (value) => `${format('.3s')(value)}tCO\u2082e`,
```

Comment:
HI! looks great, i assume `u2082e` is superscript? Could you make a utils file called `unicode.js`

Then a constant something like: 

```
export const SUPER = `\u2082e`;
```

```
import unicode from 'utils/unicode';

unitFormat: (value) => `${format('.3s')(value)}tCO${unicode.SUPER}`
```

Because I know myself and I will screw one of these Unicode up in the future :)